### PR TITLE
Bulk action status form layout improvements

### PR DESCRIPTION
**Pull request by fragalcer**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

**Comment by fragalcer** [comment]
This is how the form looks now:

image

This is how it looks in the modal:

image

File: python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py [hunk 1]
```diff
@@ -215,8 +206,16 @@ class ChangeStateForm(ChangeStatusForm, BulkActionForm):
...
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['items'].help_text = "These are the items that will be registered. " \
+                                         "Add or remove additional items with the autocomplete box."
+        self.fields['registrationDate'].help_text = "Date the registration state will be active from."
+        self.fields['cascadeRegistration'].label = "Cascade endorsement"
+        self.fields['changeDetails'].help_text = "The administrative note is a publishable statement describing the " \
+                                                 "reasons for registration."
+        # self.fields['registrationAuthorities'].widget = forms.RadioSelect()
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py]: Let's remove the commented out code.

It'd be better to move this to move it to the `ChangeStatusGenericForm` to reduce duplication.

File: python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py [hunk 2]
```diff
@@ -269,6 +266,8 @@ def __init__(self, *args, **kwargs):
             required=False,
             widget=forms.Textarea
         )
+        self.fields['items'].help_text = "These are the items that will be moved between workgroups. " \
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py]: You could define this in the base BulkAction to set `self.fields['items'].help_text` as a class attr for better abstraction.

File: python/aristotle-metadata-registry/aristotle_mdr/forms/forms.py [hunk 3]
```diff
@@ -83,7 +83,6 @@ def clean_registrationAuthorities(self):
     def clean_state(self):
         state = self.cleaned_data['state']
         state = int(state)
-        MDR.STATES[state]
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/forms.py]: Good catch!

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html [hunk 4]
```diff
@@ -5,48 +5,48 @@
...
-        </p><p>
-            By archiving this workgroup, this workgroup will longer show up in lists for users when
-            creating items (such as Object Classes or Data Elements),
-            and new posts or comments can no longer be created within the discussion area for this workgroup.
-        </p>
-        {% endif %}
+            {% if item.archived %}
+                <p>
+                    Workgroup exists to perform small discrete functions to create a well defined set of items.
+                    By reactivating a workgroup, you are stating that it the work for this group is no longer completed.
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html]:
```suggestion
                    By reactivating a workgroup, you are stating that the work for this group is no longer completed.
```

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html [hunk 5]
```diff
@@ -5,48 +5,48 @@
...
-            completed, and is a positive step to completing the standardisation process.
-        </p><p>
-            By archiving this workgroup, this workgroup will longer show up in lists for users when
-            creating items (such as Object Classes or Data Elements),
-            and new posts or comments can no longer be created within the discussion area for this workgroup.
-        </p>
-        {% endif %}
+            {% if item.archived %}
+                <p>
+                    Workgroup exists to perform small discrete functions to create a well defined set of items.
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html]: Wo
```suggestion
                    Workgroups exist to perform small discrete functions, thereby creating a well defined set of items.
```

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html [hunk 6]
```diff
@@ -5,48 +5,48 @@
...
+                    By reactivating a workgroup, you are stating that it the work for this group is no longer completed.
+                    Archiving workgroups also makes it easier for users to navigate their options when creating or searching
+                    for items.
+                </p><p>
+                It also possible to create a new workgroup, or if you are certain you
+                can reactive this workgroup by using the buttons below.
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/archive_workgroup.html]:
```suggestion
                can reactivate this workgroup by using the buttons below.
```

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/bulk_actions/lots_of_things.html [hunk 7]
```diff
@@ -2,25 +2,24 @@
...
-    <div class="modal-body">
-        <div class="alert alert-danger">
-            Woah! It looks like you are about to change a lot of things.
-            Double check whats about to happen and click 'go' when you are sure.
+    <form method="post" action="{% url 'aristotle:bulk_action' %}?next={{next}}">{% csrf_token %}
+        <input type="hidden" name="bulkaction" value="{{ bulk_action_title }}"/>
+        <div class="modal-body">
+            <div class="alert alert-danger">
+                Woah! It looks like you are about to change a lot of things.
+                Double check whats about to happen and click 'go' when you are sure.
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/bulk_actions/lots_of_things.html]:
```suggestion
                Double check what's about to happen and click 'go' when you are sure.
```

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/deprecateItems.html [hunk 8]
```diff
@@ -4,26 +4,26 @@
 
 {% block content %}
 
-<form method="post" action="{% url 'aristotle:deprecate' iid=item.id %}">{% csrf_token %}
-    <div class="modal-body">
-        <header>This form allows you to supersede older items item, by indicating a new version exists.
+    <form method="post" action="{% url 'aristotle:deprecate' iid=item.id %}">{% csrf_token %}
+        <div class="modal-body">
+            <header>This form allows you to supersede older items item, by indicating a new version exists.
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/deprecateItems.html]:
```suggestion
            <header>This form allows you to supersede older items, by indicating a new version exists.
```

**Pull request merged**